prottp.go

```diff
@@ -37,10 +40,22 @@ func Wrap(service Service) http.Handler {
 	return mux
 }
 
+var marshaler = jsonpb.Marshaler{
+	EnumsAsInts:  false,
+	EmitDefaults: false,
+	Indent:       "\t",
+	OrigName:     true,
```

Comment:
Do you think we should use `underscore_separated_names` or `camalSeparatedNames` ?